x-pack/plugins/task_manager/server/queries/mark_available_tasks_as_claimed.ts

```diff
@@ -54,15 +54,16 @@ export const IdleTaskWithExpiredRunAt: MustCondition<TermFilter | RangeFilter> =
   },
 };
 
-export const InactiveTasks: MustNotCondition<TermFilter | RangeFilter> = {
+// TODO: Fix query clauses to support this
+export const InactiveTasks: BoolClauseWithAnyCondition<any> = {
```

Comment:
I did not manage to get query clause TS types working within a my timebox window, deferring to later.